scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1065,8 +1065,8 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <property> elements.
   //
   private class NameValuePair {
-    private var name  : String = null
-    private var value : String = null
+    private var name  : String | Null = null
+    private var value : String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.